vars/dockerExecuteOnKubernetes.groovy

```diff
@@ -418,7 +418,7 @@ chown -R ${runAsUser}:${fsGroup} ."""
             echo "stash effective (excludes): ${excludes}"
         }
 
-        stash(
+        new Utils().stash(
```

Comment:
Instead of instanciating `Utils` here it might be a good idea to use the utils instance which was either received or created by `call(./.)`. Of course we need to forward that instance to the method here.

Reply:
I did not want to touch the signatures.

Reply:
Done: https://github.com/SAP/jenkins-library/pull/4380/commits/41837b0eaec45b6519995ab5afe0f0ad64ab29e0